체육복.js

```diff
@@ -0,0 +1,38 @@
+function solution(n, lost, reserve) {
+    const ALL = Array(n)
+        .fill()
+        .map((item, index) => index + 1);
+
+    let objAll = {};
+    ALL.forEach((item) => {
+        objAll[item] = 1;
+    });
```

Comment:
```suggestion
    const objAll = {};
    
    for (let i = 0; i < n; i += 1) {
        objAll[i + 1] = 1;
    }
```
forEach에 목숨걸지 않아도 됩니다.